.github/workflows/solidity.yml

```
@@ -40,16 +46,16 @@
         uses: actions/cache/save@v4
         with:
           path: contracts
-          key: ci-solidity-${{ github.ref }}
+          key: ${{ steps.set-cache-key.outputs.cache-key }}
 
       - id: forge
-        run: echo "FORGE_PATH=$(which forge)" >> $GITHUB_OUTPUT
+        run: echo "forge_path=$(which forge)" >> $GITHUB_OUTPUT
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:1:37: Double quote to prevent globbing and word splitting

(shellcheck)